core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -606,6 +614,51 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         return FixedPoint.Unsigned(_safeUintCast(oraclePrice));
     }
 
+    // Reset withdrawal request by setting the withdrawl request and withdrawl timestamp to 0.
+    function resetWithdrawalRequest(PositionData storage positionData) internal {
+        positionData.withdrawalRequestAmount = FixedPoint.fromUnscaledUint(0);
+        positionData.requestPassTimestamp = 0;
+    }
+
+    // Ensure individual and global consistency when increasing collateral balances. Returns the change to the position.
+    function incrementCollateralBalances(PositionData storage positionData, FixedPoint.Unsigned memory collateralAmount)
+        internal
+        returns (FixedPoint.Unsigned memory)
+    {
+        _addCollateral(rawTotalPositionCollateral, collateralAmount);
+        return _addCollateral(positionData.rawCollateral, collateralAmount);
+    }
+
+    // Ensure individual and global consistency when decrementing collateral balances. Returns the change to the position.
+    function decrementCollateralBalances(PositionData storage positionData, FixedPoint.Unsigned memory collateralAmount)
+        internal
+        returns (FixedPoint.Unsigned memory)
+    {
+        _removeCollateral(rawTotalPositionCollateral, collateralAmount);
+        return _removeCollateral(positionData.rawCollateral, collateralAmount);
+    }
+
+    /**
```

Comment:
I moved these functions up as before they were below private functions. `_onlyOpenState`, `_onlyPreExpiration`, `_onlyPostExpiration` & `_onlyCollateralizedPosition` were not modified.

Reply:
Good stuff